core/src/main/kotlin/org/evomaster/core/output/OutputFormat.kt

```diff
@@ -44,4 +45,23 @@ enum class OutputFormat {
 
     fun isJUnit() = this.name.contains("_junit_", true)
 
+    fun isDotnet() = this.name.startsWith("dotnet",ignoreCase = true)
+*/
+    //TODO: Remove these lines of code
+    fun isJava() = false
+
+    fun isKotlin() = false
+
+    fun isJavaScript() = false
+
+    fun isJavaOrKotlin() = isJava() || isKotlin()
+
+    fun isJUnit5() = false
+
+    fun isJUnit4() = false
+
+    fun isJUnit() = false
+
+    fun isDotnet() = true
+
```

Comment:
Sure I'll change this, thanks

Reply:
yep, should not hard-code these things ;)
it can be fine in a quick-check, but should not endup in a PR.
note that the output format can be specified in the driver configuration